model/model.go

```diff
@@ -14,6 +14,9 @@ const (
 	MinInt = (1 << bits.UintSize) / -2
 )
 
+// Domain is deprecated.nextroute engine. It is used with the router engine and store
```

Comment:
I think these comments got messed up probably with a replace or something. Do you mean to add the `Deprecated: ` note here?